css/index.css

```diff
@@ -1,3 +1,284 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 95%;
+	max-width: 880px;
+	/* TB LR */
+	margin: 0 auto;
+  }
+  
+  /* header */
+  header {
+	  display: flex;
+	  flex-wrap: wrap;
+	  align-items: center;
+	  margin-top: 35px;
+	  justify-content: space-between;
+	}
+	
+	@media(max-width: 800px) {
+		header {
+			flex-direction: column-reverse;
+			margin-top: 25px;
+		}
+		header img {
+			margin-bottom: 6%;
+		}
+	}
+
+
+
+  header nav {
+	display: flex;
+	max-width: 691px;
+	vertical-align: top;
+  }
+  
+  header nav a {
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  }
+
+	@media(max-width: 400px) {
+		header nav {
+			width: 100%;
+			height: 200px;
+			flex-direction: column;
+			justify-content: space-between;
+			align-items: center;
+		}
+		header nav a {
+			width: 100%;
+			padding: 2%;
+			margin-right: 0px;
+			border-bottom: 1px solid grey;
+			text-align: center;
+		}
+		header nav a:first-child {
+			border-top: 1px solid grey;
+		}
+	}
```

Comment:
Either it's something on my end or inside of your code, but the nav isn't stacking on top of each other. Try on your end once more to see if it's working

Reply:
Hey Phil,
So I don't know why it's doing this, but I can only seem to prompt the phone size media breakpoint to trigger if I have the dev tools open on my browser (see attached).  This doesn't surprise me, as I had the dev tools open the whole time I was working on this project and wouldn't have looked at my project without inspecting.  My media query breakpoint is set to trigger at 400px, but really doesn't unless I have dev tools open.  Do you have any idea why it's behaving like this?
![Dev Tools Error](https://user-images.githubusercontent.com/49927848/57578298-52f39400-743e-11e9-8a80-e97eed2792fb.PNG)